test/check-application

```diff
@@ -103,6 +112,22 @@ def run_e2e(verbose, image, browser, cpus, memory, sit):
         # Now actually run the tests
         selenium_address = "10.111.112.10" if selenium else ""
 
+        if "DEBUG_TEST" in os.environ:
+            try:
+                vnc_cmd = "vncviewer 127.0.0.2:{} &"
```

Comment:
And likewise, don't hardcode the IP.  `networking_selenium["forward"]["5901"], is the full `host:address` value.